src/ethereum/forks/osaka/vm/gas.py

```diff
@@ -84,6 +84,109 @@
 GAS_BLS_G2_MUL = Uint(22500)
 GAS_BLS_G2_MAP = Uint(23800)
 
+# EEST Specific Variables
+G_TX_DATA_ZERO = 4
+G_TX_DATA_NON_ZERO = 16
+
+# Transaction costs (from transactions.py)
+TX_BASE_COST = Uint(21_000)
+TX_CREATE_COST = Uint(32_000)
+TX_ACCESS_LIST_ADDRESS_COST = Uint(2_400)
+TX_ACCESS_LIST_STORAGE_KEY_COST = Uint(1_900)
+STANDARD_CALLDATA_TOKEN_COST = Uint(4)
+FLOOR_CALLDATA_COST = Uint(10)
+
+# Authorization costs (from eoa_delegation.py)
+PER_EMPTY_ACCOUNT_COST = 25_000
+PER_AUTH_BASE_COST = 12_500
+
+# Opcode Costs
+G_STOP = GAS_ZERO
+G_ADD = GAS_VERY_LOW
+G_MUL = GAS_LOW
+G_SUB = GAS_VERY_LOW
+G_DIV = GAS_LOW
+G_SDIV = GAS_LOW
+G_MOD = GAS_LOW
+G_SMOD = GAS_LOW
+G_ADDMOD = GAS_MID
+G_MULMOD = GAS_MID
+G_EXP = GAS_EXPONENTIATION
+G_SIGNEXTEND = GAS_LOW
+G_LT = GAS_VERY_LOW
+G_GT = GAS_VERY_LOW
+G_SLT = GAS_VERY_LOW
+G_SGT = GAS_VERY_LOW
+G_EQ = GAS_VERY_LOW
+G_ISZERO = GAS_VERY_LOW
+G_AND = GAS_VERY_LOW
+G_OR = GAS_VERY_LOW
+G_XOR = GAS_VERY_LOW
+G_NOT = GAS_VERY_LOW
+G_BYTE = GAS_VERY_LOW
+G_SHL = GAS_VERY_LOW
+G_SHR = GAS_VERY_LOW
+G_SAR = GAS_VERY_LOW
+G_CLZ = GAS_LOW
+G_KECCAK256 = GAS_KECCAK256
+G_ADDRESS = GAS_BASE
+G_BALANCE = GAS_WARM_ACCESS  # or GAS_COLD_ACCOUNT_ACCESS if cold
+G_ORIGIN = GAS_BASE
+G_CALLER = GAS_BASE
+G_CALLVALUE = GAS_BASE
+G_CALLDATALOAD = GAS_VERY_LOW
+G_CALLDATASIZE = GAS_BASE
+G_CALLDATACOPY = GAS_VERY_LOW
+G_CODESIZE = GAS_BASE
+G_CODECOPY = GAS_VERY_LOW
+G_GASPRICE = GAS_BASE
+G_EXTCODESIZE = GAS_WARM_ACCESS  # or GAS_COLD_ACCOUNT_ACCESS if cold
+G_EXTCODECOPY = GAS_WARM_ACCESS  # or GAS_COLD_ACCOUNT_ACCESS if cold
+G_RETURNDATASIZE = GAS_BASE
+G_RETURNDATACOPY = GAS_VERY_LOW
+G_EXTCODEHASH = GAS_WARM_ACCESS  # or GAS_COLD_ACCOUNT_ACCESS if cold
+G_BLOCKHASH = GAS_BLOCK_HASH
+G_COINBASE = GAS_BASE
+G_TIMESTAMP = GAS_BASE
+G_NUMBER = GAS_BASE
+G_PREVRANDAO = GAS_BASE
+G_GASLIMIT = GAS_BASE
+G_CHAINID = GAS_BASE
+G_SELFBALANCE = GAS_LOW
+G_BASEFEE = GAS_BASE
+G_BLOBHASH = GAS_BLOBHASH_OPCODE
+G_BLOBBASEFEE = GAS_BASE
+G_POP = GAS_BASE
+G_MLOAD = GAS_VERY_LOW
+G_MSTORE = GAS_VERY_LOW
+G_MSTORE8 = GAS_VERY_LOW
+G_SLOAD = GAS_WARM_ACCESS  # or GAS_COLD_SLOAD if cold
+G_SSTORE = GAS_WARM_ACCESS  # base cost, actual cost is more complex
+G_JUMP = GAS_MID
+G_JUMPI = GAS_HIGH
+G_PC = GAS_BASE
+G_MSIZE = GAS_BASE
+G_GAS = GAS_BASE
+G_JUMPDEST = GAS_JUMPDEST
+G_TLOAD = GAS_WARM_ACCESS
+G_TSTORE = GAS_WARM_ACCESS
+G_MCOPY = GAS_VERY_LOW
+G_PUSH0 = GAS_BASE
+G_PUSHx = GAS_VERY_LOW
+G_DUPx = GAS_VERY_LOW
+G_SWAPx = GAS_VERY_LOW
+G_LOGx = GAS_LOG
```

Comment:
```suggestion
G_PUSH = GAS_VERY_LOW
G_DUP = GAS_VERY_LOW
G_SWAP = GAS_VERY_LOW
G_LOG = GAS_LOG
```

The lowercase "x" is unusual in Python constants.